Alignment/APEEstimation/macros/DrawIteration.C

```diff
@@ -678,72 +682,80 @@ void DrawIteration::drawFinals(const std::string& xOrY){
       unsigned int iHist(1);
       for(i_hist = v_hist.begin(); i_hist != v_hist.end(); ++i_hist, ++iHist){
-		  TH1* hist((*i_hist).first);
-		  if(iHist==1){
-			  hist->Draw("e0");
-		  }
-		  else{
-			  hist->SetLineColor(iHist);
-			  hist->SetMarkerColor(iHist);
-			  hist->Draw("e0same");
-		  }
-	  }
+      TH1* hist((*i_hist).first);
+      if(iHist==1){
+        hist->Draw("e0");
+      }
+      else{
+        hist->SetLineColor(iHist);
+        hist->SetMarkerColor(iHist);
+        hist->Draw("e0same");
+      }
+    }
       
       TH1* systHist(0);
       if(systematics_){
-	const std::vector<std::string>& v_name(*i_resultHist);
-	
-	bool pixel(false);
-	bool tob(false);
+  const std::vector<std::string>& v_name(*i_resultHist);
+  
+  bool pixel(false);
+  bool tob(false);
         std::vector<std::string>::const_iterator i_name;
         for(i_name=v_name.begin(); i_name!=v_name.end(); ++i_name){
-	  const TString name((*i_name).c_str());
-	  if(name.BeginsWith("Bpix") || name.BeginsWith("Fpix")){
-	    pixel = true;
-	    break;
-	  }
-	  if(name.BeginsWith("Tob")){
-	    tob = true;
-	    break;
-	  }
-	}
-	if(pixel || tob)systHist = new TH1F("systematics", "sytematics", v_name.size(), 0, v_name.size());
-	if(pixel){
-	  if(xOrY=="x"){
-	    systHist->SetBinContent(1, 10.);
-	    systHist->SetBinContent(2, 10.);
-	    systHist->SetBinContent(3, 10.);
-	    systHist->SetBinContent(4, 10.);
-	    systHist->SetBinContent(5, 10.);
-	    systHist->SetBinContent(6, 10.);
-	    systHist->SetBinContent(9, 5.);
-	  }
-	  else if(xOrY=="y"){
-	    systHist->SetBinContent(1, 15.);
-	    systHist->SetBinContent(2, 15.);
-	    systHist->SetBinContent(3, 15.);
-	    systHist->SetBinContent(4, 20.);
-	    systHist->SetBinContent(5, 15.);
-	    systHist->SetBinContent(6, 15.);
-	    systHist->SetBinContent(9, 5.);
-	  }
-	}
-	if(tob){
-	  systHist->SetBinContent(1, 15.);
-	  systHist->SetBinContent(2, 15.);
-	  systHist->SetBinContent(3, 10.);
-	  systHist->SetBinContent(4, 10.);
-	  systHist->SetBinContent(5, 10.);
-	  systHist->SetBinContent(6, 10.);
-	  systHist->SetBinContent(7, 15.);
-	  systHist->SetBinContent(8, 10.);
-	}
+    const TString name((*i_name).c_str());
+    if(name.BeginsWith("Bpix") || name.BeginsWith("Fpix")){
+      pixel = true;
+      break;
+    }
+    if(name.BeginsWith("Tob")){
+      tob = true;
+      break;
+    }
+  }
+  if(pixel || tob)systHist = new TH1F("systematics", "sytematics", v_name.size(), 0, v_name.size());
+  if(pixel){
+    if(xOrY=="x"){
+      systHist->SetBinContent(1, 10.);
+      systHist->SetBinContent(2, 10.);
+      systHist->SetBinContent(3, 10.);
+      systHist->SetBinContent(4, 10.);
+      systHist->SetBinContent(5, 10.);
+      systHist->SetBinContent(6, 10.);
+      systHist->SetBinContent(7, 10.);
+      systHist->SetBinContent(8, 10.);
+      systHist->SetBinContent(9, 10.);
+      systHist->SetBinContent(10, 10.);
+      systHist->SetBinContent(13, 5.);
```

Comment:
please add a comment about what the bin numbers refer to (I guess it's related to layers?)

Reply:
@mteroerd Thanks for the update. Do you intend to add a comment here, too?

Reply:
Yes, this is to account for the different number of layers in the bpix. Christian made the change, but it somehow was not included in the last pull request.

Reply:
Could you then add a comment in the code for future maintainers?

Reply:
Sorry,but since I am not 100% sure for which reasons this was included in the first place (As far as I see it, it is currently not used at all, as the flag systematics_ is always false. The different macros seem to not reference it.), I would rather not write too much about it. Unfortunately, I am sick, so currently I cannot do too much research on how this works.

As far as I remember, Christian just changed it so it would be consistent with the new number of layers.

Reply:
Ok, I suggest to leave that to the next update of this code then.

Reply:
Okay, I will make sure to include this in the next update.